### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by neuromusic**
- removes roadmap (b/c now in [Projects](https://github.com/chanzuckerberg/napari-hub/projects/1)) & linked in wiki
- adds @DragaDoncila & @klai95 to team
- adds some flair

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,44 +5,60 @@
...
+We rely heavily on User Experience Research to understand the needs and challenges of the bioimaging community, identify opportunities for solutions to these challenges, and get feedback on our work. Whether you’re a bench scientist, work at an imaging core, or develop computational methods, sign up here and we’ll reach out when there’s an interview, focus group, or workshop that matches your background.
 
 ### Share your ideas
 
-Do you have ideas for new features that would help make the napari hub even better? [Join the discussion](https://github.com/chanzuckerberg/napari-hub/discussions/categories/ideas), [add your ideas](https://github.com/chanzuckerberg/napari-hub/discussions/new?category=ideas), and give feedback on other’s ideas.
+Do you have ideas for new features that would help make the napari hub even better? [Join the discussion](https://github.com/chanzuckerberg/napari-hub/discussions/categories/ideas), add your ideas, and give feedback on other’s ideas.
```
Comment (justinelarsen) [inline README.md:38]: Was removal of the link intentional?

**Review by justinelarsen — APPROVED** [review]
LGTM! 🎉

**Review by codemonkey800 — APPROVED** [review]
this is awesome, LGTM! i love the badges 🦡

**Comment by neuromusic** [comment]
@codemonkey800 I did it for you

File: README.md (hunk 1 above)
Comment (neuromusic) [reply README.md]: nope, that was an accident

**Review by LCObus — APPROVED** [review]
woohoo! thanks for the updates, keeping us fresh!

**Pull request merged**